### ISSUE TITLE
Mirroring appear on the 8 LED matrix

### ISSUE THREAD

**Issue by mikekgr** [Qrome/marquee-scroller#131]
Dear Sirs,
I bought 8 LED matrix ( in 3 blocks: 3 + 3 + 2 ) from this supplier:
https://www.electrodragon.com/product/dot-matrix-chain-display-kit-max7219-v2/
Following the documentation, I connected my ESP8266 WEMOS the 3 signals plus power, to the input of the LED matrix that I mentioned above.
Despite my many tries, the best I can do is what the following video shoes:
https://photos.app.goo.gl/X9wCdc9r1VW2e9Lp8
Using the:
int ledRotation = 1;
at settings.h

Can someone to help me please ?
As far as I can tell the using library: 
https://github.com/markruys/arduino-Max72xxPanel
seems not to handle well my LED matrix...
If this is an unavoided obstacle, can we use for this project the Bellow library that is working fine with my LED matrix?
https://github.com/MajicDesigns/MD_Parola

By the way, I like this project very much and I would like to congratulate the owner of this project.

Thanks and Best Regards,
Mike Kranidis

**Comment by xtianpaiva** [comment]
Could you try this patch? https://github.com/Qrome/marquee-scroller/pull/132/commits/fd8ac4b56d337bf771e7e7b9ab90ba491a6f3eda it worked for me, PR sent.

**Comment by mikekgr** [comment]
> Could you try this patch? [fd8ac4b](https://github.com/Qrome/marquee-scroller/commit/fd8ac4b56d337bf771e7e7b9ab90ba491a6f3eda) it worked for me, PR sent.

Yes tomorrow I will try it and I will let you know the result. Many thanks

**Comment by mikekgr** [comment]
> Could you try this patch? [fd8ac4b](https://github.com/Qrome/marquee-scroller/commit/fd8ac4b56d337bf771e7e7b9ab90ba491a6f3eda) it worked for me, PR sent.

Dear @xtianpaiva  , again thanks for your help try. I did what you suggested but the result is not the desired. There is not improvement unfortunately, still in "mirror mode". And this can be done only using:
int ledRotation =1; @ Settings.h
Please see the New video using your patching but with ledRotation=1 
https://photos.app.goo.gl/nyfifjroJLDJKpCS7
If you need, I can make a new video using the default ledRotation =3

Best Regards,
Mike Kranidis

**Comment by mikekgr** [comment]
Anyone that can help me? Please.

**Comment by mikekgr** [comment]
@Qrome Any helping hand to my problem?
Thanks
